src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -227,8 +227,10 @@ const preFetchQuery = gql`
 				loanAmount
 				status
 				use
+				fundraisingPercent @client
```

Comment:
We need these to calculate the percent for the share message. I don't think this will impact performance on this query

Reply:
I think this may re-introduce some excessive logging that we recently cleaned up. This local query has data requirements that oddly enough, aren't available on all loans...

Can you please move it out of the prefetch?

Reply:
@eddieferrer What I'm suggesting is to move just the client query out into it's own process and to check for it's data dependencies before calling it please.